src/AFQMC/config.h

```diff
@@ -28,8 +28,6 @@
 #include "multi/memory/fallback.hpp"
 
 #include "Utilities/TimerManager.h"
-#include "AFQMC/Utilities/myTimer.h"
-extern myTimer Timer;
```

Comment:
Where is the definition of "Timer" correspond to this extern?
AFQMCFactory.cpp?

Reply:
Yes. I believe this is legacy code and myTimer is no longer used globally in the AFQMC code, only for debugging with definite scope. I had to remove the extern to avoid name clash with Timer.h

Reply:
Although given the build errors this assumption was wrong. Let me fix these.

Reply:
I mean if the global one is not needed any more. The definition needs to be removed as well.
In addition, please fix the CI failures.